doc/dev.md

```diff
@@ -98,9 +103,15 @@ Return generic "not found/failed" error message to user, log the original error.
 
 2. SCM errors
 Some of these can only be fixed by the user who is calling the method by interacting with UI elements (usually course teacher).
-Example: if a github organization cannot be found, one of the possible issues causing this behavior is disabled third party access. As a result, the requested organization cannot be seen by the application. If a GitHub repo or team cannot be found, they could have been manually deleted from GitHub. Only the current user can remedy the situation, and it is most useful to inform them about the issue in detail and offer a solution.
-Another example: sometimes GitHub interactions take too long and the request context can be canceled by GitHub. This is not an application failure, and neither developer team, nor user can actively do anything to correct it. But it is still useful to inform the user that the action must be repeated at later time to succeed.
-Return a custom error with method name and original error for logging, and a custom message string to be displayed to user.
+
+Examples: 
+- If a GitHub organization cannot be found, one of the possible issues causing this behavior not having installed the GitHub application on the organization. As a result, the requested organization cannot be seen by QuickFeed. 
+- If a GitHub repository or team cannot be found, they could have been manually deleted from GitHub. Only the current user can remedy the situation, and it is most useful to inform them about the issue in detail and offer a solution.
+
+Sometimes GitHub interactions take too long and the request times out, or is otherwise cancelled by GitHub.
+In these cases the error is usually ephemeral in nature, and the action should be repeated at later time. This should be communicated to the end user.
+
+Return a custom error with detailed information for logging, and a custom error message to user.
 
 3. Access control errors
 Return generic "access denied" error message to user.
```

Comment:
... to **the** user.

Reply:
Not sure, but I think maybe this should be indented to be below the `A` in `Access` to make it clear that it belongs...